extension/src/Template/TaskRunnerItem.ts

```diff
@@ -1,7 +1,22 @@
+import { Position, Uri } from "vscode";
+
 export interface TaskRunnerItem {
   description: string;
   command: string;
+  arguments?: CommandParameter[];
```

Comment:
I think we have to go for `any[]` here...
![image](https://user-images.githubusercontent.com/23032645/163733196-0e18f2d1-3dfb-42f8-b0ae-3f54285045f9.png)

Reply:
`unknown[]` maybe?